Jess branch

Jessica-L opened the pull request:
Here's my code finally for plant storage (includes: main plant database, facade module to interface with db called plantStorage.java and plant.java which is used to create and access plant objects.  Next tasks: I'll work with Calvin on implementing the search functionality and the back end code for the user's plant list.

Jessica-L committed:
added plant storage modules + plant image files

Jessica-L committed:
corrected naming convention of the 4 pant images

Jessica-L committed:
Modified plant storage modules

Added getter methods to be used by search functionality; if plant name
not in db, returns null so front end can deal with this case properly,
i.e. display msg to user on screen like "Plant not found.  Please try
your search again."

Commit:
checkin

Commit:
resolved merge conflicts

Commit:
Merge branch 'noah-branch' into jess-branch

ghost commented:
this is not good to merge yet, for some reason that i do not know the "add plant" button is now gone.

Jessica-L commented:
what module was this "add plant" button located in?

Regards,

Jessica Lynch
303-870-8395
www.linkedin.com/in/jessicalynchcsci/
Undergraduate -- B.S. in Computer Science, Software Engineering
University of Colorado -- Boulder, CO

Intern II (Part-time) -- Firmware, Integration & Test Process Engineering
Seagate -- Longmont, CO

On Wed, Apr 8, 2015 at 9:58 PM, TheNoar notifications@github.com wrote:

> this is not good to merge yet, for some reason that i do not know the "add
> plant" button is now gone.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/CalvineHicks/Florasphere/pull/1#issuecomment-91109657
> .

Commit:
resolved merge conflicts

Commit:
Merge branch 'noah-branch'

Commit:
Merge branch 'jess-branch'

CalvineHicks committed:
fixed errors in order to run code on my computer

Jessica-L committed:
cleaned up spaces

Jessica-L committed:
Merge remote-tracking branch 'origin/jess-branch' into jess-branch

Conflicts:
	FlorasphereNEW/app/src/main/java/com/example/florasphere/Plant.java

Jessica-L committed:
got database to work with front end

Jessica-L committed:
renamed plant image to adhere to naming convention

CalvineHicks committed:
adding search functionality

CalvineHicks committed:
Merge branch 'jess-branch' of https://github.com/CalvineHicks/Florasphere into jess-branch

Commit:
changes

Commit:
pull master resolve merge conflicts

Commit:
pull jess-branch fixed merge conflicts

Jessica-L committed:
userPlantList trial

Commit:
merging plant object

Commit:
merged plant object

Commit:
fixed picture thingy sortof

Jessica-L committed:
deleted commented code in FlorasphereListActivity

Commit:
merged confilcts fixed

Jessica-L committed:
final db fix plus demo set up for user plantlist

dbList in FlorasphereListActvity will provide a nice plantlist showing
all four plants for our demo next week thursday --> dbList will be
replaced with userPlantList (singleton) AFTER search for plant
funtionality is done.

Jessica-L committed:
final db fix

Jessica-L committed:
Merge remote-tracking branch 'origin/jess-branch' into jess-branch

Conflicts:

	FlorasphereNEW/app/src/main/java/com/example/florasphere/SearchSubmissionActivity.java
	FlorasphereNEW/app/src/main/res/layout/searchsubmission.xml

Commit:
added watering functionality

Commit:
added search results list and adding plant functionalisty

Commit:
merging master

Jessica-L committed:
Merge remote-tracking branch 'origin/master' into jess-branch

Conflicts:

	FlorasphereNEW/app/src/main/java/com/example/florasphere/FlorasphereListArrayAdaptor.java
	FlorasphereNEW/app/src/main/java/com/example/florasphere/Plant.java

	FlorasphereNEW/app/src/main/java/com/example/florasphere/PlantDatabase.java

	FlorasphereNEW/app/src/main/java/com/example/florasphere/PlantStorage.java

	FlorasphereNEW/app/src/main/java/com/example/florasphere/SearchSubmissionActivity.java
	FlorasphereNEW/app/src/main/res/layout/searchsubmission.xml

Jessica-L committed:
update submission xml file

Commit:
merged master

Commit:
merged master and fixes

Jessica-L committed:
Merge pull request #2 from CalvineHicks/user_plist_trial

User plist trial needs to be merged with jess-branch

Jessica-L committed:
water droplet images + functionality

Jessica-L committed:
added florasphere logo + cleaned up images

Jessica-L committed:
new ic_launcher app icon
